Makefile

```diff
@@ -198,6 +198,14 @@ build-query:
 build-collector:
 	CGO_ENABLED=0 installsuffix=cgo go build -o ./cmd/collector/collector-$(GOOS) $(BUILD_INFO) ./cmd/collector/main.go
 
+.PHONY: build-ingester
+build-ingester:
+	CGO_ENABLED=0 installsuffix=cgo go build -o ./cmd/ingester/ingester-$(GOOS) $(BUILD_INFO) ./cmd/ingester/main.go
+
+.PHONY: build-ingester-linux
```

Comment:
This was added by @zoidbergwill but there's no `build-collector-linux` nor a `build-agent-linux`, so I see 2 options:
1. I remove this and we recommend using `GOOS=linux make build-ingester`
2. I add one for every binary

Reply:
Ahhh, I see now, only the all in one has a `-linux` job. Recommending `GOOS=linux make build-ingester` would definitely be good enough. I thought I'd found some other ones that did the same, my bad.

Reply:
Hum I missed that one as well.

Reply:
It's used by `./scripts/travis/build-all-in-one-image.sh`, but I can add that cleanup to the PR